### ISSUE TITLE
Fix for Shadowlands

### ISSUE THREAD

**Issue by spiralofhope** [spiralofhope/_DevPad#15]
It doesn't work..

To investigate.

<details>
 <summary>Click to expand</summary>

```
Message: Interface\AddOns\_DevPad.GUI\_DevPad.GUI.Dialog.lua:200: attempt to call method 'SetBackdrop' (a nil value)
Time: Sat Oct  3 05:26:07 2020
Count: 1
Stack: Interface\AddOns\_DevPad.GUI\_DevPad.GUI.Dialog.lua:200: attempt to call method 'SetBackdrop' (a nil value)
[string "@Interface\AddOns\_DevPad.GUI\_DevPad.GUI.Dialog.lua"]:200: in function `New'
[string "@Interface\AddOns\_DevPad.GUI\_DevPad.GUI.List.lua"]:10: in main chunk
[string "=[C]"]: in function `LoadAddOn'
[string "@Interface\AddOns\_DevPad\_DevPad.lua"]:531: in function `?'
[string "@Interface\FrameXML\ChatFrame.lua"]:5076: in function `ChatEdit_ParseText'
[string "@Interface\FrameXML\ChatFrame.lua"]:4740: in function `ChatEdit_SendText'
[string "@Interface\FrameXML\ChatFrame.lua"]:4776: in function `ChatEdit_OnEnterPressed'
[string "*:OnEnterPressed"]:1: in function <[string "*:OnEnterPressed"]:1>

Locals: self = <table> {
 OnSizeChanged = <function> defined @Interface\AddOns\_DevPad.GUI\_DevPad.GUI.Dialog.lua:108
 WindowOnMouseWheel = <function> defined @Interface\AddOns\_DevPad.GUI\_DevPad.GUI.Dialog.lua:141
 AddTitleButton = <function> defined @Interface\AddOns\_DevPad.GUI\_DevPad.GUI.Dialog.lua:63
 OnMouseUp = <function> defined @Interface\AddOns\_DevPad.GUI\_DevPad.GUI.Dialog.lua:90
 ScrollButtonOnClick = <function> defined @Interface\AddOns\_DevPad.GUI\_DevPad.GUI.Dialog.lua:145
 StickColor = <table> {
 }
 Unpack = <function> defined @Interface\AddOns\_DevPad.GUI\_DevPad.GUI.Dialog.lua:171
 OnMouseDown = <function> defined @Interface\AddOns\_DevPad.GUI\_DevPad.GUI.Dialog.lua:85
 New = <function> defined @Interface\AddOns\_DevPad.GUI\_DevPad.GUI.Dialog.lua:190
 Pack = <function> defined @Interface\AddOns\_DevPad.GUI\_DevPad.GUI.Dialog.lua:160
 StickyFrames = <table> {
 }
 ResizeOnMouseUp = <function> defined @Interface\AddOns\_DevPad.GUI\_DevPad.GUI.Dialog.lua:77
 ScrollBarOnValueChanged = <function> defined @Interface\AddOns\_DevPad.GUI\_DevPad.GUI.Dialog.lua:151
 StickTolerance = 16
 ScrollFrameOnScrollRangeChanged = <function> defined @Interface\AddOns\_DevPad.GUI\_DevPad.GUI.Dialog.lua:113
 SetVerticalScrollToCoord = <function> defined @Interface\AddOns\_DevPad.GUI\_DevPad.GUI.Dialog.lua:26
 ResizeOnMouseDown = <function> defined @Interface\AddOns\_DevPad.GUI\_DevPad.GUI.Dialog.lua:73
 ControlOnEnter = <function> defined @Interface\AddOns\_DevPad.GUI\_DevPad.GUI.Dialog.lua:19
 ScrollFrameOnVerticalScroll = <function> defined @Interface\AddOns\_DevPad.GUI\_DevPad.GUI.Dialog.lua:132
 NewButton = <function> defined @Interface\AddOns\_DevPad.GUI\_DevPad.GUI.Dialog.lua:40
}
Name = "_DevPadGUIList"
Frame = _DevPadGUIList {
 0 = <userdata>
}
(*temporary) = nil
(*temporary) = _DevPadGUIList {
 0 = <userdata>
}
(*temporary) = <table> {
 bgFile = "Interface\TutorialFrame\TutorialFrameBackground"
 tileSize = 32
 edgeFile = "Interface\TutorialFrame\TutorialFrameBorder"
 tile = true
 edgeSize = 32
 insets = <table> {
 }
}
(*temporary) = "attempt to call method 'SetBackdrop' (a nil value)"
ResizeTexture = "Interface\AddOns\_DevPad.GUI\Skin\ResizeGrip"
```
</details>

**Comment by spiralofhope** [comment]
Fixed in retail 8.3.7.1, classic 1.13.1

Turns it it was a simple library issue.

Note that https://github.com/spiralofhope/_DevPad.GUI/issues/29 remains.

**Comment by spiralofhope** [comment]
Turns it it was a simple library issue.
